[2022-08-02 10:00:09,529] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: Demo.records_saving manual__2022-08-02T09:59:45.179821+00:00 [queued]>
[2022-08-02 10:00:09,549] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: Demo.records_saving manual__2022-08-02T09:59:45.179821+00:00 [queued]>
[2022-08-02 10:00:09,550] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-02 10:00:09,550] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2022-08-02 10:00:09,551] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-02 10:00:09,586] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): records_saving> on 2022-08-02 09:59:45.179821+00:00
[2022-08-02 10:00:09,593] {standard_task_runner.py:52} INFO - Started process 4048 to run task
[2022-08-02 10:00:09,601] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'Demo', 'records_saving', 'manual__2022-08-02T09:59:45.179821+00:00', '--job-id', '203', '--raw', '--subdir', 'DAGS_FOLDER/demo.py', '--cfg-path', '/tmp/tmpo55e9a88', '--error-file', '/tmp/tmp9b1e0znn']
[2022-08-02 10:00:09,602] {standard_task_runner.py:80} INFO - Job 203: Subtask records_saving
[2022-08-02 10:00:09,759] {task_command.py:371} INFO - Running <TaskInstance: Demo.records_saving manual__2022-08-02T09:59:45.179821+00:00 [running]> on host 31353b71d6cb
[2022-08-02 10:00:10,017] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=Demo
AIRFLOW_CTX_TASK_ID=records_saving
AIRFLOW_CTX_EXECUTION_DATE=2022-08-02T09:59:45.179821+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-08-02T09:59:45.179821+00:00
[2022-08-02 10:00:10,055] {base.py:68} INFO - Using connection ID 'postgres_demo' for task execution.
[2022-08-02 10:00:10,067] {dbapi.py:231} INFO - Running statement: CREATE TABLE IF NOT EXISTS public."Files"
                            (
                                id integer NOT NULL,
                                name character(80)[] COLLATE pg_catalog."default" NOT NULL,
                                size bigint,
                                CONSTRAINT "Files_pkey" PRIMARY KEY (id)
                            ), parameters: None
[2022-08-02 10:00:10,090] {base.py:68} INFO - Using connection ID 'postgres_demo' for task execution.
[2022-08-02 10:00:10,101] {dbapi.py:231} INFO - Running statement: INSERT INTO public."Files"(
                                id, name, size)
                                VALUES (%()s, %()s, %()s);, parameters: {'id': 0, 'name': 10, 'size': 100}
[2022-08-02 10:00:10,103] {taskinstance.py:1909} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 171, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 189, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/demo.py", line 62, in save_records
    VALUES (%()s, %()s, %()s);""",parameters={"id":i,"name":i+10,"size":i+100})
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/hooks/dbapi.py", line 211, in run
    self._run_command(cur, sql_statement, parameters)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/hooks/dbapi.py", line 233, in _run_command
    cur.execute(sql_statement, parameters)
KeyError: ''
[2022-08-02 10:00:10,147] {taskinstance.py:1420} INFO - Marking task as FAILED. dag_id=Demo, task_id=records_saving, execution_date=20220802T095945, start_date=20220802T100009, end_date=20220802T100010
[2022-08-02 10:00:10,190] {standard_task_runner.py:97} ERROR - Failed to execute job 203 for task records_saving (''; 4048)
[2022-08-02 10:00:10,214] {local_task_job.py:156} INFO - Task exited with return code 1
[2022-08-02 10:00:10,428] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
